### ISSUE TITLE
possible to show no-undef ?

### ISSUE THREAD

**Issue by leviwheatcroft** [ezhlobo/eslint-plugin-react-pug#11]
Not really sure if this is possible or if this is the right module, but it would be super helpful if eslint could show undefined variables used in the pug template.

**Comment by ezhlobo** [comment]
@leviwheatcroft thank you for mentioning this. Yes, this is right place and it's on my roadmap. I can't promise you a deadline, but I'll keep you updated ;).

**Comment by ezhlobo** [comment]
@leviwheatcroft hey, this notification is just for keeping you updated with progress.

I've started and implemented 'proof of concept'. Remaining part is to do refactoring and extend README. It also requires some time because I would like to test that rule in my current project, so probably it will be merged in 1 week.

You can test it as well right now by adding explicit dependency in package.json:
```
"eslint-plugin-react-pug": "https://github.com/ezhlobo/eslint-plugin-react-pug.git#rule/no-undef"
```

BTW, now it looks not very good because it marks the whole template as an error, so it might be tricky to find not defined variable in big template.

![screen shot 2018-02-02 at 21 02 07](https://user-images.githubusercontent.com/974552/35747722-5d695e82-085c-11e8-85d5-67b2b2b841db.png)

**Comment by leviwheatcroft** [comment]
nice! I'll take a look

**Comment by ezhlobo** [comment]
@leviwheatcroft hey, I've just published [0.2 version](https://github.com/ezhlobo/eslint-plugin-react-pug/releases/tag/v0.2.0) which supports this rule. If you have not tried it yet, then now just increase version number in your package.json ;). 

Please, re-open this issue if that update does not resolve your concern.

**Comment by leviwheatcroft** [comment]
awesome. thanks. love your work mate.
